Packs/ApiModules/Scripts/OktaApiModule/README.md

```diff
@@ -0,0 +1,14 @@
+The Okta API Module handles the OAuth 2.0 authentication process when sending requests.
+
+To use the API module, add the `from OktaApiModule import *  # noqa: E402` line of code after the `main` function definition, but before it's being called, as shown in the example below.
+
+```python
+def main():
+    ...
+
+
+from OktaApiModule import *  # noqa: E402
+
+if __name__ in ('__main__', '__builtin__', 'builtins'):
+    main()
+```
```

Comment:
I just copy-pasted the README description from other API modules (and modified it).
Should I just remove the README / leave it empty then?

Reply:
to be sure, put the import line at the top, and see if something is wrong,
if all is good, i think you can leave it empty, as for Packs/ApiModules/Scripts/AWSApiModule/README.md
because this is an internal script for our use

Reply:
You are right, seems like putting it at the top works without any issues.
I've removed the README file